packages/dotnet-sdk/src/TeamsFx.Test/TeamsFxTests.cs

```diff
@@ -1,38 +1,40 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
Are these 2 `using`s are not necessary any more? Since we have enabled implicit using in csproj file.
You could use "Analyze and Code Cleanup" -> "Run Code Cleanup" to auto format codes, I've configured the rules for it.

Reply:
Yes, you are right. These can be removed. I'll run the code cleanup